### DISCUSSION TITLE
[Feature] Install instruction for existing postgresql server

### DISCUSSION THREAD

**Discussion by jeanpaulrh** [immich-app/immich#2272] [Q&A] [answered]
### The feature

Hi,
I would like to avoid having multiple postgresql instances running on my machine for every container which needs a db, so I would like to use a db inside my existing postgresql container. Is it possible? Is there a doc that explains how to install immich with an existing postgresql server?
Thanks

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment, marked as answer]
It doesn't need anything special, just configure the `DB_*` variables in the .env file to point at your existing database.

**Reply by jeanpaulrh** [reply]
Thanks. I'm sorry I've asked. I never used a .env file for my docker, and I didn't read its content while browsing the documentation.
Thanks again.
